flask_restful/__init__.py

```diff
@@ -79,6 +79,7 @@ def __init__(self, app=None, prefix='',
         self.catch_all_404s = catch_all_404s
         self.url_part_order = url_part_order
         self.errors = errors or {}
+        self.errorhandlers = []
```

Comment:
I thought so, but later decided to change to a loop so that it can to handle subclasses as well (following flask implementation)